src/clients/inst_autosetup.rb

```diff
@@ -345,6 +345,9 @@ def main
         )
 	# failed relnotes download is not fatal, ignore ret code
 	WFM.CallFunction("inst_download_release_notes")
+        # Do not start it in second installation stage again.
+        # Registration will already be called in the first stage.
+        Profile.remove_sections("suse_register")
```

Comment:
Just a question: Will this also avoid installing `yast2-registration` (and it's dependencies) to the target system?

Reply:
Not really, but at least AutoYaST does not select the package. anymore. I am assuming it will be installed by the dependencies now..

Reply:
OK, thanks!